install_rock.sh

```diff
@@ -1,6 +1,14 @@
-tmp="$(wget -O - https://github.com/cogneco/rock/releases/latest | grep -o '\<download/rock_.*deb\>')"
-wget https://github.com/cogneco/rock/releases/$tmp
-echo ${tmp##*/}
-dpkg-deb -x ${tmp##*/} .
-mv usr/bin/rock .
-mv usr/lib/fancy_backtrace.so .
+#!/bin/bash
+if [ "$(uname -o)" == "Msys" ]; then
+	tmp="$(wget -O - https://github.com/cogneco/rock/releases/latest | grep -o '\<download/rock_.*zip\>')"
+	wget https://github.com/cogneco/rock/releases/$tmp
+	echo ${tmp##*/}
+	7z e ${tmp##*/}
+else
+	tmp="$(wget -O - https://github.com/cogneco/rock/releases/latest | grep -o '\<download/rock_.*deb\>')"
+	wget https://github.com/cogneco/rock/releases/$tmp
+	echo ${tmp##*/}
+	# dpkg-deb -x ${tmp##*/} .
+	# mv usr/bin/rock .
+	# mv usr/lib/fancy_backtrace.so .
```

Comment:
Fixed it as soon as I saw the Circle CI test fail. It was not intentional but a mistake to leave the comment there.